### DISCUSSION TITLE
0.8.1-alpha.2

### DISCUSSION THREAD

**Discussion by Tyh2001** [FightingDesign/fighting-design#249] [Announcements]
详细更新日志请参考 [CHANGELOG.md](https://github.com/FightingDesign/fighting-design/blob/master/CHANGELOG.md)。

Please refer to [CHANGELOG.en-US.md](https://github.com/FightingDesign/fighting-design/blob/master/CHANGELOG.en-US.md) for details.

<hr /><em>This discussion was created from the release <a href='https://github.com/FightingDesign/fighting-design/releases/tag/0.8.1-alpha.2'>0.8.1-alpha.2</a>.</em>

**Comment by keer-tea** [comment]
作者真的厉害，向大佬学习:scream:
